.github/workflows/ci.yml

```diff
@@ -21,9 +21,6 @@ jobs:
         include:
           # Test supported ROS 2 distributions
           # https://docs.ros.org/en/rolling/Releases.html
-          # NOTE: Humble does not work on the `ros2` branch, so it is tested in its own branch.
```

Comment:
I think you want to keep this comment, but amend it with "Jazzy and Humble" right?